_data/simple-icons.json

```
@@ -11339,6 +11339,11 @@
             "hex": "000000",
             "source": "https://docs.paperspace.com/img/ps-logo-light.svg"
         },
+        {
+            "title": "Paradox Interactive",
+            "hex": "000000",
+            "source": "https://www.paradoxinteractive.com/favicons/safari-pinned-tab.svg"
```

I believe we should just link to the homepage here.
```suggestion
            "source": "https://www.paradoxinteractive.com"
```
And also we should make sure your SVG, based on this pinned tab icon, matches https://www.paradoxinteractive.com/images/logo/logo-platypus-white.svg (the header icon on the main site).